docker/pg_tests/scripts/tests/partial_restore_test.sh

```diff
@@ -66,6 +66,7 @@ else
 fi
 
 if psql -t -c "SELECT COUNT(*) FROM tbl;" -d third -A 2>&1 | grep -q "is not a valid data directory"; then
+  /tmp/scripts/drop_pg.sh
```

Comment:
Let's instead do a clean-up in the common tests run pipeline? So it would guarantee that nobody will forget to add this cleanup later. And (maybe) in the separate PR?

Reply:
I don't mind doing it in a separate PR